src/Raven.Server/Documents/AI/AiGen/ChatCompletionClient.cs

```diff
@@ -0,0 +1,169 @@
+﻿using System;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Net.Http.Json;
+using System.Runtime.InteropServices;
+using System.Text.Json;
+using System.Text.Json.Nodes;
+using System.Threading.Tasks;
+
+namespace Raven.Server.Documents.AI.AiGen;
```

Comment:
AiGen or GenAi?

Reply:
The name should be `GenAI` all `AiGen` should be replaced to `GenAI`